projection/core/src/main/scala/com/lightbend/lagom/internal/projection/ProjectionRegistryActor.scala

```diff
@@ -158,14 +174,12 @@ class ProjectionRegistryActor extends Actor with ActorLogging {
             }
         }
 
-      requestedStatusLocalCopy = remotelyChanged
+      requestedStatusLocalCopy = replicatedEntries
```

Comment:
renamed var for clarity: a `Changed` message may also be triggered by local `Update` operations. The previous name of the `val` was misleading.